app/views/profiles/_certificate.html.slim

```diff
@@ -0,0 +1,2 @@
+- cache certificate do
+  .card-certificate style="background-image: url(#{certificate.cover_url})"
```

Comment:
@vyper check the last commit, I've added this option as you suggested, but in local I can't download the pdf (check the error below), can you test there?

![screen shot 2017-01-03 at 13 53 31](https://cloud.githubusercontent.com/assets/4116980/21613420/530f2cc8-d1bc-11e6-9733-a6d8ed024401.png)